main.py

```diff
@@ -1,10 +1,53 @@
-import init_django_orm  # noqa: F401
+import json
+
+from django.db.utils import IntegrityError
 
+import init_django_orm  # noqa: F401
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player in data:
+        race_details = data[player]["race"]
+        skill_details = race_details["skills"]
+        current_guild = data[player]["guild"]
+
+        if not Race.objects.filter(name=race_details["name"]).exists():
+            Race.objects.create(
+                name=race_details["name"],
+                description=race_details["description"]
+            )
+        current_race = Race.objects.get(name=race_details["name"]).id
```

Comment:
You can use get_or_create:
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create